components/dashboard/src/teams/sso/OIDCClients.tsx

```diff
@@ -29,16 +33,44 @@ type OIDCClientsListProps = {
     clientConfigs: OIDCClientConfig[];
 };
 const OIDCClientsList: FC<OIDCClientsListProps> = ({ clientConfigs }) => {
+    const { toast } = useToast();
     const [showCreateModal, setShowCreateModal] = useState(false);
+    const [activateModalConfigId, setActivateModalConfigID] = useState("");
+
+    const verifyClient = useVerifyClient({
+        onSuccess: (configId: string) => {
+            toast(
+                <span>
+                    Single sign-on configuration was successfully verified.{" "}
+                    <LinkButton onClick={() => setActivateModalConfigID(configId)}>Activate configuration</LinkButton>.
```

Comment:
**suggestion**: I'd drop the period from the link button to help it stand out and create better visual balance, as the period looks a bit out of place.

```suggestion
                    <LinkButton onClick={() => setActivateModalConfigID(configId)}>Activate configuration</LinkButton>
```